app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -14,6 +14,7 @@ CONFIG_COMP_SRC_IPC4_FULL_MATRIX=y
 CONFIG_PM=y
 CONFIG_PM_DEVICE=y
 CONFIG_PM_DEVICE_RUNTIME=y
+CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE=y
```

Comment:
do we need this?

Reply:
We don't, we do need to trigger pm hooks.